Cargo.toml

```diff
@@ -45,10 +49,15 @@ cookie-factory = "0.3.1"
 hostname = "0.3.0"
 
 [dev-dependencies]
+rmp-serde = "0.14"
+rmpv = { version = "0.4.5", features = ["with-serde"] }
+serde_yaml = "0.8"
+serde-pickle = "0.6"
```

Comment:
Do we really need to list these under dev-dependencies too?

Reply:
We do not need to; The alternative is that the tests will only run when `--features extra_formats` is passed on the command line. In the interest of avoiding regressions I was looking for a way to make sure the tests run without special knowledge of what features to enable.

Afaik it is not possible to indicate that this feature should be enabled for tests in cargo.toml. This is issue 2911 in cargo, which is where I found the suggestion to do it this way.

https://github.com/rust-lang/cargo/issues/2911#issuecomment-415998960